src/test/java/org/redquark/algorithms/datastructures/lists/CircularLinkedListTest.java

```diff
@@ -72,6 +72,8 @@ public void testAddAfterNode() {
         assertEquals(11, testCircularLinkedList.head.next.next.next.next.next.data);
         assertEquals(6, testCircularLinkedList.head.next.next.next.next.next.next.data);
         assertThrows(IllegalArgumentException.class, () -> testCircularLinkedList.addAfterNode(20, 12));
+        testCircularLinkedList.addAfterNode(100,10);
```

Comment:
Can we add space between *100,* and *10*.

Reply:
Updated the PR!